<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalSearchWidget</class>
 <widget class="QWidget" name="GlobalSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>539</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout_search">
   <item row="0" column="0">
    <widget class="QLabel" name="label_fuzzy">
     <property name="text">
      <string>Fuzzy matching</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_fuzzy</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_fuzzy">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_separators">
     <property name="text">
      <string>Separators</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_separators</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_separators">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_decay">
     <property name="text">
      <string>Memory behavior</string>
     </property>
     <property name="buddy">
      <cstring>slider_decay</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>Most frequently used. Remember everything. </string>
       </property>
       <property name="text">
        <string>MFU</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_decay">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The decay the weights past activation get. Without decay the search remembers everything and behaves in a most frequently used fashion, while a high decay will prefer recently used item. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="sliderPosition">
        <number>55</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>Most recently used. Forget fast. </string>
       </property>
       <property name="text">
        <string>MRU</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_prioritizePerfectMatch">
     <property name="text">
      <string>Prioritize perfect matches</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_prioritizePerfectMatch</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_prioritizePerfectMatch">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_globalHandlers">
     <property name="text">
      <string>Global search handlers</string>
     </property>
     <property name="buddy">
      <cstring>listWidget_globalHandlers</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_fallbackHandlers">
     <property name="text">
      <string>Fallback handlers</string>
     </property>
     <property name="buddy">
      <cstring>listWidget_fallbackHandlers</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QListWidget" name="listWidget_globalHandlers">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QListWidget" name="listWidget_fallbackHandlers">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
